contents/handbook/strategy/business-model.md

```diff
@@ -29,12 +27,12 @@ If the wider community contributes a new feature that isn't already a source-ava
 
 When PostHog makes a new feature, we ask ourselves two questions:
 
-1. Who is the likely type buyer of this feature?
+1. Is this feature for Creation, Collaboration or Compliance?
 1. Would this feature help more users find and use PostHog?
 
-If the likely buyer is an individual contributor, the feature will be open source. Otherwise, if the likely buyer is a manager, director or executive, it will be source available. The exception to this is if the feature will significantly help the community to increase. For example, initially we planned "multiple users" as a feature for the source-available version. However, we decided that having multiple users would help the community to grow, which benefits everyone disproportionately.
+If the likely use case is use by an individual contributor, the feature will be open source. Otherwise, it will be source available. The exception to this is if the feature will significantly help the community to increase. For example, initially we planned "multiple users" as a feature for the source-available version. However, we decided that having multiple users would help the community to grow, which benefits everyone disproportionately.
```

Comment:
`source-available version` isn't a term we use anywhere else. `paid functionality` perhaps?